docs/how-to/build-policies/abac/temporary_role.mdx

```diff
@@ -0,0 +1,145 @@
+---
+sidebar_position: 6
+title: Temporary Role Example
+---
+
+Below you'll find how to implement temporary role assignments using ABAC Condition Sets in Permit.io, with a step-by-step implementation guide.
```

Comment:
Maybe explain what's temporary role assignment is, and give an example use case for it